content/en/blog/2020/custom-istio-dns-name/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Customizing Istio DNS stub names"
+subtitle: Changing or adding stub domains to make ServiceEntry hostnames routable
+description: Customizing global or region hostnames as declared in ServiceEntry(s) for routability.
+publishdate: 2020-05-05
+attribution: "Christian Posta (Solo.io)"
+keywords: [istio, dns, mTLS, coredns]
+---
+
+Istio can be run across multiple clusters and has the necessary configuration APIs to support [shared control plane](/docs/setup/install/multicluster/shared/) or a non-shared, [replicated control model](/docs/setup/install/multicluster/gateways). The motivation for deploying across multiple clusters has been covered excellently by fellow Istio contributor [Vadim Eisenberg](https://twitter.com/vadimeisenberg) in the [Multi-Mesh Deployments for Isolation and Boundary Protection](/blog/2019/isolated-clusters/) blog.
+
+One aspect of multi-cluster functionality enabling services to communicate with each other is service discovery. In an Istio replicated control-plane multi-cluster setup, you create [ServiceEntry](/docs/reference/config/networking/service-entry/)s in one cluster to point to services running in another cluster. For example:
```

Comment:
I found the first sentence of this paragraph to be a little bit hard to follow, partly because of the sentence structure and partly because it relies so heavily on multi-syllable words. The sentence is perfectly grammatical, but I wonder if you could make it a little easier to read, maybe by changing it to something like this:

"Multi-cluster Istio lets services communicate with each other, which lets services discover each other."

Or maybe the end of that sentence should be something like "...which allows for service discovery"? I suppose "which enables service discovery" would also be OK in this context.

We usually mildly recommend against using "functionality" and "enable" in this sort of context:

https://developers.google.com/style/word-list#functionality

https://developers.google.com/style/word-list#enable

Those words aren't a big problem, but they do sometimes make docs a little harder to read.

But in this case, "enables" might end up being the best compromise between being brief and being accurate.